ckanext/aircan/views.py

```
@@ -0,0 +1,165 @@
+                logs = None
+            aircan_status["logs"] = logs
+
+            error_val = (
+                json.loads(aircan_status.get("error", "{}"))
+                if aircan_status.get("error")
+                else None
+            )
+
+            try:
+                aircan_status["error"] = json.loads(error_val) if error_val else {}
+            except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)